lib/scss/inc/_variables.scss

```diff
@@ -41,7 +41,8 @@ $sidebar-width: 350px;
 $sidebar-bg: #151515;
 $doclink-header-color: $white;
 $doclink-color: rgba($white, 0.75);
-$doclink-hover-color: $white;
+$doclink-hover-color: rgba($white, 0.875);
```

Comment:
LGTM except if @rushenn  wants to have a more subtle color for a selected document.